src/content/configuration/output.md

```diff
@@ -728,7 +728,7 @@ There is no need to change it.
 
 `boolean`
 
-Tell webpack to remove a module from the module instance cache (`require.cache`) if it throws an exception when it is `require`d.
+Tell webpack to remove a module from the module instance cache (`require.cache`) if it throws an exception when it is `required`.
```

Comment:
isnt `require` specifically wrap to imply a command here? or is there a `required` directive?

Reply:
Sorry, about that I think my understanding was not correct.
We can close the PR.